manim/__init__.py

```diff
@@ -8,9 +10,25 @@
 __version__ = importlib_metadata.version(__name__)
 
 
+import sys
+
 # Importing the config module should be the first thing we do, since other
 # modules depend on the global config dict for initialization.
 from ._config import *
+
+# Workaround to set the renderer passed via CLI args *before* importing
+# Manim's classes (as long as the metaclass approach for switching
+# between OpenGL and cairo rendering is in place, classes depend
+# on the value of config.renderer).
+for ind, arg in enumerate(sys.argv):
+    if arg.startswith("--renderer"):
```

Comment:
Imo as long as `--use_opengl_renderer` and `--use_webgl_renderer` are usable arguments, they should be handled here

Reply:
I also agree: 9cfdc3fa34fd7dbffe7f7d9bf95ccdffd0912c7b